pex/testing.py

```diff
@@ -135,7 +137,7 @@ def make_project(
         os.path.join(name, "my_module.py"): 'def do_something():\n  print("hello world!")\n',
         os.path.join(name, "package_data/resource1.dat"): 1000,
         os.path.join(name, "package_data/resource2.dat"): 1000,
-    }
+    }  # type: Dict[str, Union[str, int]]
```

Comment:
This is the best we can do with type casting. We can't use `cast()` because we can't use values like `Dict` at runtime.

Reply:
See above, but this is untrue, you just need to pass the type to cast as a string.